drivers/storage/portworx/preflight.go

```diff
@@ -208,6 +208,14 @@ func (u *preFlightPortworx) RunPreFlight() error {
 		logrus.Infof("runPreflight: running pre-flight with existing PX-StoreV2 param, hard fail check enabled")
 	}
 
+	// Remove phone-home cm volume mount, does not exist yet
+	for i, volumeMount := range preflightDS.Spec.Template.Spec.Containers[0].VolumeMounts {
+		if volumeMount.Name == "ccm-phonehome-config" {
+			preflightDS.Spec.Template.Spec.Containers[0].VolumeMounts = append(preflightDS.Spec.Template.Spec.Containers[0].VolumeMounts[:i],
+				preflightDS.Spec.Template.Spec.Containers[0].VolumeMounts[i+1:]...)
+			break
+		}
+	}
```

Comment:
is this covered by existing UTs?

Reply:
No not yet... Adding one now.

Reply:
Instead of this volume mount getting added and then removed, can you explore not adding it in the first place itself? This will improve debugability if someone  is trying to figure out the spec that is used for pre-flight where they just need to look at one function (the getter for the pod spec).

Reply:
Not now.. I just spent time.. .adding a unit test.